modules/apps/cache/cache.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
```

Comment:
this provider should not be placed under apps dir.
I think it should be separated into:
- `modules/core/project/cache.go`
- `modules/core/org/cache.go`

Reply:
add unit-test for cache.